### PR TITLE
Add formatter

### PR DESCRIPTION
**Pull request by ngthanhvinh**

### CODE REVIEW

**Review by guruprerana — DISMISSED** [review]

**Comment by ngkan** [comment]
The only important thing here is the format style and not astyle right? And if so can we just make a .md for it instead?

**Comment by ngthanhvinh** [comment]
The formatter helps us not to care much about styles since it automatically formats the code.

File: src/trading-platform-pp.pro [hunk 1]
```diff
@@ -11,7 +11,8 @@ CONFIG += c++11
 SOURCES += \
     api/api.cpp \
     main.cpp \
-    gui/mainwindow.cpp
+    gui/mainwindow.cpp \
+    style.astylerc
```
Comment (ngkan) [inline src/trading-platform-pp.pro]: I don't think this is necessary.

Comment (ngthanhvinh) [reply src/trading-platform-pp.pro]: This is necessary to activate the formatter in Qt Creator

Comment (ngkan) [reply src/trading-platform-pp.pro]: We should use DISTFILES variable instead. Take a look at https://doc.qt.io/qtcreator/creator-beautifier.html section 11, "Select the Use file defined in project files"

Comment (ngthanhvinh) [reply src/trading-platform-pp.pro]: Have you tried to put it in DISTFILES? It only works for me when I put it in SOURCES.

Comment (ngkan) [reply src/trading-platform-pp.pro]: It works well for me. Everytime I edit and save a file, Qt Creator automatically formats the file.

**Comment by ngthanhvinh** [comment]
I have moved the style file to DISTFILES. It works for me now!

**Review by ngkan — APPROVED** [review]

**Review by guruprerana — APPROVED** [review]

**Pull request merged**